lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -30,6 +30,17 @@ def self.checkout_and_pull(branch)
         return false
       end
 
+      def self.cut_release_branch(branch_name)
+        if branch_exists?(branch_name)
+          UI.message("Branch #{branch_name} already exists. Skipping creation.")
```

Comment:
It seems like if I run `git checkout -b cleanup/git-helpers` in this branch, it already fails with `fatal: A branch named 'cleanup/git-helpers' already exists.`

Might be overthinking this, but could we use the `create_branch` in place of this?

Reply:
Ditto https://github.com/wordpress-mobile/release-toolkit/pull/206#discussion_r571265425 – seems this comment was made on an outdated diff